### ISSUE TITLE
Failing to upload without displaying errors when periods are in geotiff filename

### ISSUE THREAD

**Issue by jackreid** [samapriya/geeup#52]
First off, thanks for the absolutely wonderful utility. This thing has saved me so many hours over the past several months.

### Summary: 

When using `geeup upload` on a collection of geotiff files that include periods in their filenames, the upload fails but no error is provided. I do not know if this is because of syntax restrictions in geeup or in GEE.

### Desired Outcome

Properly upload files that have periods in the filename. If this is not possible because of restrictions on the GEE side of things, then at least present an informative error rather than processing through all the images.

### Steps to Reproduce

1. Have a geotiff with periods in the filename, other than part of the file extension. Example: `/home/jackreid/Downloads/temp/santaClaraNightlights/temp/geotiffs/VNP46A2.A2020002.h05v05.001.2021053031537.tif` from [NASA VIIRS](https://ladsweb.modaps.eosdis.nasa.gov/missions-and-measurements/products/VNP46A2/)
2. Generate metadata using `geeup getmeta`
3. Upload using `geeup upload`
4. See standard processing text. Eg:
```
Collection XXXXXX already exists
Total of 1 assets remaining : Total of 0 already in collection with 0 associated tasks running or submitted
2022-10-04 16:12:04 INFO Processing image 1 out of 1 : /filepath/VNP46A2.A2020004.h05v05.001.2021053040601.tif
```
5. Processing completes. Check GEE via the browser or API and no image has been uploaded, nor has any task been started. This can be confirmed with `geeup quota`

### Current Workaround

I used a terminal command to replace all the (non file extension) periods with underscores, then re-generated metadata and attempted uploading. This seems to work fine. 

### System Details
Operating System: Ubuntu 22.04.1 LTS
Python Version: 3.10.6
geeup Version: Not sure how to figure this out actually. I reinstalled it just recently, so I believe it is the current version.

**Comment by samapriya** [comment]
Unfortunately, this is how GEE ingest is designed currently, you can try the same uploading a single image with multiple periods and GEE will not allow it. I will include a better error message in the future to avoid situations where the program completes without any logs

**Comment by jackreid** [comment]
Makes sense. Thanks for adding the error message, as that will save future folks some time in troubleshooting.

**Comment by samapriya** [comment]
Now fixed in v0.6.0 thanks for being patient :)
